test/unit/app/messaging/validate-inbound-message.test.js

```diff
@@ -0,0 +1,80 @@
+import { validateStatusMessageRequest } from '../../../../app/messaging/validate-inbound-message.js'
+
+const mockSetBindingsLogger = jest.fn()
+const mockedLogger = {
+  setBindings: mockSetBindingsLogger
+}
+
+const validInputMessage = {
+  crn: 1050000003,
+  sbi: 105000003,
+  agreementReference: 'IAHW-ABCD-3FGH',
+  claimReference: 'FUDC-N87C-PIN5',
+  claimStatus: '2',
+  dateTime: new Date()
+}
+
+describe('validateStatusMessageRequest', () => {
+  afterEach(() => {
+    jest.resetAllMocks()
```

Comment:
Whilst this repo is still small and doesn't have a huge number of tests, would be nice to put `restoreMocks: true` and `resetMocks: true` into the `jest.config.cjs` file so we don't have to keep adding these everywhere, (or forgetting to). https://jestjs.io/docs/configuration#resetmocks-boolean

Reply:
Deal